CesiumGltfContent/src/GltfUtilities.cpp

```diff
@@ -269,13 +274,55 @@ GltfUtilities::parseGltfCopyright(const CesiumGltf::Model& gltf) {
 
   Buffer& destinationBuffer = gltf.buffers[0];
 
+  std::vector<bool> keepBuffer(gltf.buffers.size(), false);
+  keepBuffer[0] = true;
+
   for (size_t i = 1; i < gltf.buffers.size(); ++i) {
     Buffer& sourceBuffer = gltf.buffers[i];
+
+    // Leave intact any buffers that have a URI and no data.
+    if (sourceBuffer.uri && sourceBuffer.cesium.data.empty()) {
+      keepBuffer[i] = true;
+      continue;
+    }
```

Comment:
Meshopt fallback buffers should also be left intact:

https://github.com/KhronosGroup/glTF/blob/main/extensions/2.0/Vendor/EXT_meshopt_compression/README.md#fallback-buffers